README.md

```diff
@@ -4,34 +4,58 @@ Experimental support for VS Code Server in NixOS. The NodeJS by default supplied
 
 ## Installation
 
+### Flake
 ```nix
 {
-  imports = [
-    (fetchTarball "https://github.com/msteen/nixos-vscode-server/tarball/master")
-  ];
-
-  services.vscode-server.enable = true;
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixos-vscode-server.url ="github:iosmanthus/nixos-vscode-server/add-flake";
```

Comment:
Shouldn't this point to msteen/nixos-vscode-server instead of your fork?

Reply:
Sorry, I forgot to change those lines.